cli/pcluster/commands.py

```diff
@@ -545,7 +538,7 @@ def ssh(args, extra_args):  # noqa: C901 FIXME!!!
     :param args: pcluster CLI args
     :param extra_args: pcluster CLI extra_args
     """
-    pcluster_config = PclusterConfig(file_sections=[ALIASES])  # FIXME it always search for the default config file
+    pcluster_config = PclusterConfig()  # FIXME it always search for the default config file
```

Comment:
I don't like we are loading the `[cluster default]` content, because it will fail if you have a wrong parameter in that section.

We should at least set `fail_on_error=False`.

Reply:
Done